polaris-react/src/components/Choice/Choice.tsx

```diff
@@ -40,24 +38,18 @@ export function Choice({
   labelHidden,
   helpText,
   onClick,
-  onMouseOut,
```

Comment:
Weary of removing onMouseOut and onMouseOver as props on `Choice` 
Considering there are [usages of this component ](https://grokt.shopify.io/results?q=%3CChoice+lang%3Atsx+-r%3APolaris) outside of Polaris.